.github/workflows/build.yml

```diff
@@ -0,0 +1,35 @@
+name: build
+
+on: [push, pull_request]
+
+jobs:
+  build-with-pip:
+    name: ${{ matrix.os }}/py${{ matrix.python-version }}/pip
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 45
+    env:
+      MPLBACKEND: agg
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        python-version: [3.7, 3.8]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Display versions
+        run: python -V; pip -V
+      - name: Install depedencies and package
+        shell: bash
+        run: pip install -U -e .'[tests]'
+      - name: Run tests
+        run: pytest --cov=pyxem --pyargs pyxem
+      - name: Generate line coverage
+        if: ${{ matrix.os == 'ubuntu-latest' }}
+        run: coverage report --show-missing
+      - name: Upload coverage to Coveralls
+        if: ${{ matrix.os == 'ubuntu-latest' }}
```

Comment:
As mentioned, the only thing (I think [99% sure]) missing to get this reporting to work is adding `relative_files = True` under `[coverage:run]` in `setup.cfg` for `coverage` to find.

Reply:
yes, I think that's right, but I didn't want to create a setup.cfg for the 0.12.x branch just to have to merge it into the 0.13.x branch with a conflict (we have a setup.cfg there now for other things)